streamly-lz4.cabal

```diff
@@ -58,7 +58,7 @@ library
   build-depends:       base                >= 4    && < 5
                      , fusion-plugin-types >= 0.1  && < 0.2
                      , exceptions          >= 0.8  && < 0.11
-                     , streamly            >= 0.8  && < 0.9
+                     , streamly            >= 0.8.1 && < 0.9
```

Comment:
This bound has to be `== 0.8.1` because we are using internal modules.

Can you make a revision of streamly-lz4 on hackage to fix the version bound of streamly to `== 0.8.0`, it would be broken now.

Reply:
Yeah.